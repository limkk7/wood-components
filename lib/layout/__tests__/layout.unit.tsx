import * as renderer from "react-test-renderer";
import * as React from "react";
import { Layout, Header, Content } from "wood-components";

describe("layout", () => {
  it("是个 layout", () => {
    const json = renderer
      .create(
        <Layout>
          <Header>header</Header>
          <Content>content</Content>
        </Layout>
      )
      .toJSON();
    expect(json).toMatchSnapshot();
  });
});
