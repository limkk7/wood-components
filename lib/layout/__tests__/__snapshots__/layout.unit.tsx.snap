// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout 是个 layout 1`] = `
<div
  className="w-ui-layout"
>
  <div
    className="w-ui-layout-header"
  >
    header
  </div>
  <div
    className="w-ui-layout-content"
  >
    content
  </div>
</div>
`;
